CMakeLists.txt

```diff
@@ -89,13 +89,13 @@ set(DATADIR "share")
 endif(WIN32)
 set(SHAREDIR ${DATADIR})
 
-if (NOT APPLE)
-    set(PLUGINDIR "${CMAKE_INSTALL_FULL_LIBDIR}/${APPLICATION_SHORTNAME}/plugins" CACHE STRING "Extra path to look for Qt plugins like for VFS. May be relative to binary.")
-else()
-    # Inside the .app bundle
-    set(PLUGINDIR "../PlugIns" CACHE STRING "Extra path to look for Qt plugins like for VFS. May be relative to binary.")
```

Comment:
I assume this is because we do not build the full bundle inside the build folder anymore? Or why is this not needed anymore?